server/src/completion.ts

```diff
@@ -203,11 +203,11 @@ export function asCodeActions(details: tsp.CompletionEntryDetails, filepath: str
     };
 }
 
-export function asDetail({ displayParts, source }: tsp.CompletionEntryDetails): string | undefined {
+function asDetail({ displayParts, sourceDisplay, source: deprecatedSource }: tsp.CompletionEntryDetails): string | undefined {
     const result: string[] = [];
-    const importPath = asPlainText(source);
-    if (importPath) {
-        result.push(`Auto import from '${importPath}'`);
+    const source = sourceDisplay || deprecatedSource;
+    if (source) {
+        result.push(`Auto import from '${asPlainText(source)}'`);
```

Comment:
Any cases where this change has observable effect?

Reply:
I haven't found any, no. It should be backwards compatible with old versions of typescript since I'm still using `source`, but I didn't test explicitly.

Reply:
Well, it looks like a good change anyway as `sourceDisplay` is supposed to be the human-readable one anyway.